lib/fastlane_core/runner.rb

```diff
@@ -0,0 +1,64 @@
+module FastlaneCore
+  # this module is meant to be private to this lib
+  module Runner
+    def self.run(command, &block)
+      select_runner_impl.call(command, &block)
+    end
+
+    private
+
+    def self.select_runner_impl
```

Comment:
`private` (on line 8) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.